### PR TITLE
Playbooks for all scenarios

### PR DESCRIPTION
**Pull request by anton-sidelnikov**

### CODE REVIEW

File: playbooks/lb_monitoring.yaml [hunk 1]
```diff
@@ -58,16 +42,16 @@
...
       ignore_errors: yes
 
-- name: Start reporting for scenario
+- name: Start test
   hosts: lb_host
   become: yes
   vars_files:
-    - "./vars/csm_controller.yml"
-    - "./vars/lb_monitoring.yml"
+    - ./vars/csm_controller.yml
```
Comment (lego963) [inline playbooks/lb_monitoring.yaml]: `.yml` to `.yaml` pls here and 50line

**Review by lego963 — CHANGES_REQUESTED** [review]

File: playbooks/as_monitoring.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        state: absent
+      ignore_errors: yes
+
+    - name: Run load generation script  # noqa 301 305
+      shell: >
+        start-stop-daemon -Sbmvp /tmp/cpu_load.pid -x /usr/bin/python3
+        /tmp/cpu_load.py -- --source /tmp/load_levels.dat --interval 60 --ncpus 2
+
+    - name: Changing perm of "check_cpu_load_status.sh", adding "+x"
+      file: dest=/tmp/check_cpu_load_status.sh mode=a+x
```
Comment (outcatcher) [inline playbooks/as_monitoring.yaml]: Why single-line format is used here?

File: playbooks/as_monitoring.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    - name: Schedule load script status check
+      cron:
+        name: Check status of CPU load script
+        minute: "*/59"
+        job: /tmp/check_cpu_load_status.sh
```
Comment (outcatcher) [inline playbooks/as_monitoring.yaml]: `/tmp/check_cpu_load_status.sh` can be made a variable

File: playbooks/hdd_monitoring.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      sata: /dev/vdb
+      sas: /dev/vdc
+      ssd: /dev/vdd
+  tasks:
+    - name: Cron job for create big files on all devices every 15 minutes
+      cron:
+        name: "Creating big file on {{ item.value }}"
+        minute: "*/5"
+        job: >
+          /bin/dd if=/dev/urandom of="/mnt/{{ item.key }}/bigfile.txt"
```
Comment (outcatcher) [inline playbooks/hdd_monitoring.yaml]: It's not actually a text file, maybe name it just `bigfile`?

File: playbooks/iscsi_monitoring.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  vars:
+    device: /dev/sda
+    mount_point: /mnt/scsi
+  tasks:
+    - name: Cron job for create big files on SCSI device every 5 minutes
+      cron:
+        name: Creating big file on iSCSI device
+        minute: "*/5"
+        job: >
+          /bin/dd if=/dev/urandom of="{{ mount_point }}/bigfile.txt"
```
Comment (outcatcher) [inline playbooks/iscsi_monitoring.yaml]: Sam, as before: it's not a text file

File: playbooks/iscsi_monitoring.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+      cron:
+        name: Reading big files from iSCSI device
+        minute: "*/2"
+        job: /sbin/hdparm -tT "{{ device }}"
+
+    - name: Schedule connection test
+      cron:
+        name: Check status of iSCSI connection
+        minute: "*/1"
+        job: /tmp/iscsi_connection_status.sh
```
Comment (outcatcher) [inline playbooks/iscsi_monitoring.yaml]: Maybe, path to the script should be a variable

File: playbooks/lb_monitoring.yaml [hunk 7]
```diff
@@ -1,14 +1,14 @@
 ---
 - name: Prepare controller variables
-  import_playbook: prepare_variables.yml
+  import_playbook: prepare_variables.yaml
   vars:
-    scenario_name: 'csm_controller lb_monitoring'
+    scenario_name: csm_controller lb_monitoring
```
Comment (outcatcher) [inline playbooks/lb_monitoring.yaml]: Maybe use `>` mutiline string?

File: playbooks/rds_backup_monitoring.yaml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+          -o PasswordAuthentication=no
+
+- name: Start test
+  hosts: rds_backup_host
+  become: yes
+  vars_files:
+    - ./vars/rds_backup_monitoring.yaml
+  vars:
+    cloud_name: "{{ lookup('env', 'OS_CLOUD') }}"
+    cloud_config: /etc/openstack
```
Comment (outcatcher) [inline playbooks/rds_backup_monitoring.yaml]: `cloud_name` and `cloud_config` should be part of the inventory

**Review by outcatcher — CHANGES_REQUESTED** [review]
To be discussed: is there a need for `cron` jobs

**Review by Polina-Gubina — DISMISSED** [review]

File: playbooks/hdd_monitoring_writing.yaml [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+- name: Start test
+  hosts: hdd_hosts
+  become: yes
+  vars:
+    devices:
+      sata: /dev/vdb
+      sas: /dev/vdc
+      ssd: /dev/vdd
+  tasks:
+    - name: Create big files on all devices every 5 minutes
```
Comment (outcatcher) [inline playbooks/hdd_monitoring_writing.yaml]: It's not running every 5 minutes :)

Comment (anton-sidelnikov) [reply playbooks/hdd_monitoring_writing.yaml]: this part to remember when we include this in apimon)

**Review by lego963 — DISMISSED** [review]

**Review by Polina-Gubina — DISMISSED** [review]

**Review by outcatcher — DISMISSED** [review]

**Review by outcatcher — DISMISSED** [review]

**Review by lego963 — DISMISSED** [review]

**Comment by anton-sidelnikov** [comment]
recheck

**Review by outcatcher — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**